specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,653 @@
+          },
+          "Get integration user names over security connector scope": {
+            "$ref": "./examples/integrations/GetSecurityConnectorIntegrationUserNames_example.json"
+          },
+          "Get integration user names over management group scope": {
+            "$ref": "./examples/integrations/GetManagementGroupIntegrationUserNames_example.json"
+          }
+        },
+        "tags": [
+          "Integrations"
+        ],
+        "description": "Get user integration names for the requested scope by integrationId",
+        "operationId": "IntegrationUserNames_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/IntegrationId"
+          },
+          {
+            "$ref": "#/parameters/NamePrefixBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationUserNames"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IntegrationsList": {
+      "type": "object",
+      "description": "Page of a integrations list",
+      "properties": {
+        "value": {
+          "description": "Collection of integrations in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Integration"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "Integration": {
+      "type": "object",
+      "description": "Integration over a given scope",
+      "properties": {
+        "properties": {
+          "description": "Properties of an integration",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/IntegrationProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "IntegrationProperties": {
+      "type": "object",
+      "description": "Describes properties of an integration",
+      "properties": {
+        "tenantId": {
+          "description": "The tenantId (GUID)",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Display name of the integration",
+          "type": "string"
+        },
+        "isDisabled": {
+          "description": "Defines whether the integration is active/inactive",
+          "type": "string"
```

Comment:
>string [](http://example.com/codeflow?start=19&length=6)

It should be boolean, right?

Reply:
@visingla-ms This is what we did, the previous reviewer asked to change it to string....

Reply:
We better loosely type the parameter to prevent future breaking changes
This applies to other comments too, regarding Booleans and Enums

Reply:
The review comment might have been to change it to enum based string, instead of just string.

Reply:
Changed it to bool - done.